### ISSUE TITLE
Question about rotation in annotation files

### ISSUE THREAD

**Issue by RealYuWang** [tudelft-iv/view-of-delft-dataset#104]
Hi there,

First of all, thank you for your great work! 
I'm trying to figure out how the rotation in the annotation file was obtained. According to your description, **"We define the rotation of objects around the LiDAR’s negative vertical ( -Z) axis"** So I drew a BEV graph and the reference system is LiDAR. Is $\theta$ in it the rotation?

<img width="823" alt="草稿 jpeg" src="https://github.com/user-attachments/assets/6c4fdd0e-600f-414f-ac8f-8520ff878ea3">

**Comment by andraspalffy** [comment]
No, what you have plotted is "alpha" in Kitti's terminology. Rotation is the orientation of the bbox itself, regardless of its realtive observation angle (what you plotted as theta).
See more here:
https://github.com/tudelft-iv/view-of-delft-dataset/blob/main/docs/ANNOTATION.md
